xapian-applications/omega/templates/clicklog

```diff
@@ -0,0 +1,3 @@
+$httpheader{Status,302 Found}
+$httpheader{Location,$cgi{URL}}
+$log{clicks.log,$cgi{QUERYID}$.$cgi{DOCID}}
```

Comment:
While the values you pass for `QUERYID` and `DOCID` won't contain characters which need quoting in a CSV file (e.g. commas, double quotes, newlines), someone malicious could call this template passing values for `QUERYID` and `DOCID` which do and that could easily break the format of the log file.  Either the values need vetting before `$log` is used to store them (and bad values discarded), or they need to be quoted with `$csv`.